cypress.config.ts

```diff
@@ -7,8 +7,8 @@ export default defineConfig({
     video: false,
     screenshotOnRunFailure: false,
     setupNodeEvents(on, config) {
-      // modify config values examples
-      // config.defaultCommandTimeout = 10000
+       config.defaultCommandTimeout = 20000
+       config.responseTimeout = 20000
```

Comment:
add the semicolon at the end of each line

Reply:
Resolved!